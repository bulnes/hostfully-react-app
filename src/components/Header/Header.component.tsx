export function Header() {
  return (
    <header className="text-center p-10">
      <h1 className="font-bold text-4xl mb-4">Hostfully</h1>
      <p className="text-3xl">Front-end Test</p>
    </header>
  );
}
