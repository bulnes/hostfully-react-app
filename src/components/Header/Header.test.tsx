import { render } from "@testing-library/react";
import { Header } from "./Header.component";

describe("Component: Header", () => {
  it("should be defined", () => {
    expect(Header).toBeDefined();
  });

  it("should render correctly", () => {
    const wrapper = render(<Header />);
    expect(wrapper).toBeDefined();
  });

  it("should render title correctly", () => {
    const wrapper = render(<Header />);
    expect(wrapper.getByText("Hostfully")).toBeDefined();
  });

  it("should render subtitle correctly", () => {
    const wrapper = render(<Header />);
    expect(wrapper.getByText("Front-end Test")).toBeDefined();
  });

  it("should be represented by the snapshot", () => {
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
  });
});
