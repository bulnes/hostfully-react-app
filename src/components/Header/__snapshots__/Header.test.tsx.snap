// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Header should be represented by the snapshot 1`] = `
<div>
  <header
    class="text-center p-10"
  >
    <h1
      class="font-bold text-4xl mb-4"
    >
      Hostfully
    </h1>
    <p
      class="text-3xl"
    >
      Front-end Test
    </p>
  </header>
</div>
`;
